virt-v2v/scripts/rhel/run/network_config_util.sh

```diff
@@ -8,13 +8,13 @@ UDEV_RULES_FILE="${UDEV_RULES_FILE:-/etc/udev/rules.d/70-persistent-net.rules}"
 
 # Check if mapping file does not exist
 if [ ! -f "$V2V_MAP_FILE" ]; then
-    echo "File $V2V_MAP_FILE does not exist. Exiting."
+    echo "File $V2V_MAP_FILE does not exist. Exiting." >&2
```

Comment:
can we have a function like

```
err() { echo "$@" >&2 ; }
```

Reply:
done

Reply:
changed to log file descriptor and redirect to stdout as @mnecas suggested

tests:
dash virt-v2v/scripts/rhel/run/network_config_util_test